.github/workflows/r.yml

```diff
@@ -282,6 +282,12 @@ jobs:
           restore-keys: |
             r-${{ matrix.config.rtools }}-ccache-mingw-${{ matrix.config.arch }}-${{ hashFiles('cpp/src/**/*.cc','cpp/src/**/*.h)') }}-
             r-${{ matrix.config.rtools }}-ccache-mingw-${{ matrix.config.arch }}-
+      - name: Install CMake via MSYS2
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: UCRT64
+          update: true
+          install: mingw-w64-ucrt-x86_64-cmake
```

Comment:
Hmm. We can't use built-in CMake in `windows-2019`?

Reply:
Maybe you can, and you just have to pass in the right path-to-cmake to the build script?